plugins/analytics/public/virustotal_api.py

```diff
@@ -0,0 +1,105 @@
+from core.analytics import OneShotAnalytics
+from core.observables import Hostname, Ip, Url
+from core.entities import Company
+import urllib
+import json
+
+
+BASE_IP_URL = 'https://www.virustotal.com/vtapi/v2/ip-address/report'
+BASE_URL_URL = 'https://www.virustotal.com/vtapi/v2/url/report'
+BASE_IP_PARAMS = {'ip': None, 'apikey': None}
+BASE_URL_PARAMS = {'resource': None, 'apikey': None}
+VT_QUERY = 'Virustotal Query'

```

Comment:
Why set a global `BASE_IP_PARAMS` (same goes for `BASE_URL_PARAMS`)? If you get the API from the settings attribute, you can just dynamically construct this in line 42 when it's needed.

Also a maybe rename `VT_QUERY` to something more explicit? I'm not sure how it's used in this case.